client/tests/src/main/scala/sttp/tapir/client/tests/ClientBasicTests.scala

```diff
@@ -133,7 +135,7 @@ trait ClientBasicTests { this: ClientTests[Any] =>
     }
 
     // the fetch API doesn't allow bodies in get requests
-    if (!platformIsScalaJS) {
+    if (!platformIsScalaJS && !platformIsScalaNative) {
```

Comment:
hm if this test fails because it's a GET with a body, maybe we should change it to a correct one, where we perform a POST :) or at least create an issue to do so